### ISSUE TITLE
MALF-AI Sound Effect is way too loud.

### ISSUE THREAD

**Issue by Owai-Seek** [Skyrat-SS13/Skyrat13#3226]
## Round ID:

2003

## Date and Time of Issue:
Unknown

## Reproduction:

Have RDS, or have a MALF AI toggle Doomsday. It rips your eardrums apart.

**Comment by calibraptor** [comment]
Yeah aimalf.ogg is literally just earrape. I'll see about replacing it.
